src/ucs/config/parser.c

```diff
@@ -855,6 +857,162 @@ void ucs_config_help_array(char *buf, size_t max, const void *arg)
     array->parser.help(buf + strlen(buf), max - strlen(buf), array->parser.arg);
 }
 
+int ucs_config_sscanf_allow_list(const char *buf, void *dest, const void *arg)
+{
+    ucs_config_allow_list_t *field  = dest;
+    void *temp_field                = NULL;
+    const ucs_config_array_t *array = arg;
+    char *str_dup, *token, *saveptr;
+    int ret;
+    unsigned i;
+
+    str_dup = strdup(buf);
```

Comment:
ucs_strdup

Reply:
I copied this code from `ucs_config_sscanf_array`, which uses `strdup`.
So, if we change only here to `ucs_strdup` it will be inconsistent. Or we change there as well. What do you say?